src/components/ActionControls/ActionControls.scss

```diff
@@ -4,9 +4,21 @@
     .ba-action-controls {
         background: white;
         color: $fours;
+        margin: 0;
         white-space: normal;
     }
 
+    .ba-annotation-input-container {
+        border-top: none;
+        width: 252px; // Hard-coded width to solve layout issues
```

Comment:
What kind of layout issues were you seeing?

Reply:
There's only a max width set on the dialog so when you're adding the first comment to an annotation the width is off
![image](https://user-images.githubusercontent.com/3299001/47583485-7e80b800-d90c-11e8-88de-c8e522d1388e.png)